php_phongo.c

```diff
@@ -2331,8 +2331,22 @@ PHP_MINFO_FUNCTION(mongodb)
 	php_info_print_table_header(2, "mongodb support", "enabled");
 	php_info_print_table_row(2, "mongodb version", MONGODB_VERSION_S);
 	php_info_print_table_row(2, "mongodb stability", MONGODB_STABILITY_S);
+#ifdef HAVE_MONGOC_GET_VERSION
+	{
+	char *tmp;
+
+	spprintf(&tmp, 0, "%d.%d.%d", bson_get_major_version(), bson_get_minor_version(), bson_get_micro_version());
+	php_info_print_table_row(2, "libmongoc headers version", MONGOC_VERSION_S);
+	php_info_print_table_row(2, "libmongoc library version", mongoc_get_version());
+	php_info_print_table_row(2, "libbson headers version", BSON_VERSION_S);
+	php_info_print_table_row(2, "libbson library version", tmp);
```

Comment:
Now that 1.2.0-rc0 of libbson and libmongoc have `bson_get_version()` and `mongoc_get_version()` functions, respectively, should this be changed? The `*_get_version()` functions are both wrappers for returning the `*_VERSION_S` constants, so I'm not sure why the headers and library rows can't use the same.

The `ssprintf()` definitely appears problematic, as it'd miss the "rc0" suffix, which is present in the constant (and returned by the `*_get_version()` function).